src/zjs_net_config.c

```diff
@@ -152,40 +148,46 @@ static int str2bt_addr_le(const char *str, const char *type, bt_addr_le_t *addr)
     return 0;
 }
 
-void zjs_init_ble_address()
+static void ble_set_address(char *ble_addr)
 {
     static const struct bt_storage storage = {
         .read = zjs_ble_storage_read,
         .write = NULL,
         .clear = NULL,
     };
 
-    if (str2bt_addr_le(default_ble, "random", &id_addr) < 0) {
+    if (str2bt_addr_le(ble_addr, "random", &id_addr) < 0) {
         ERR_PRINT("bad BLE address string\n");
         return;
     }
 
-    // set the top two bits since Zephyr always sets them anyway, so we will
-    //   print the actual value the user will see
+    // set the top two bits since Zephyr always sets them anyway, for exaxmple,
+    //   if you try to set 00:F9:50:21:43:4A, Zephyr is only set it to
+    //   F1:F9:50:21:43:4A, so we will print the actual value the user will see
```

Comment:
Run-on sentence and the example isn't right. How about leave it and add:

// for example, 11:22:33:44:55:66 will change to D1:22:33:44:55:66.